Solutions/CohesitySecurity/Tools/get_role_definition_id.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/zsh
+
+SCRIPT=$(realpath "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+cd "$SCRIPTPATH"
+
+#
+# Description: This script retrieves the role definition ID for a given role name.
+
+role_name="$1"
+
+# Get role definitions
+role_definitions=$(az role definition list --output json)
+
+# Filter role definition by roleName and extract the id
+filtered_role_definition_id=$(echo $role_definitions | jq -r ".[] | select(.roleName == \"$role_name\") | .name")
```

Comment:
Fyi, all the 'az' command by default have an in-built jmespath available to filter based any attributes. So, technically, the role_definitions and filtered_role_definition_id be combined.

Reply:
just updated, will test again later, to see whether it works.